R/likelihood.R

```diff
@@ -1,93 +1,104 @@
-#' Estimate the (log) likelihood for observed branching processes
+#' Estimate the log-likelihood/likelihood for observed branching processes
 #'
 #' @inheritParams simulate_summary
-#' @param chains Vector of sizes/lengths of transmission chains.
-#' @param nsim_obs Number of simulations if the likelihood is to be
-#' approximated for imperfect observations.
-#' @param log Logical; Should the results be log-transformed? (Defaults
-#' to TRUE).
+#' @inheritParams offspring_ll
+#' @param nsim_obs Number of simulations if the log-likelihood/likelihood is to
+#' be approximated for imperfect observations.
+#' @param log Logical; Should the log-likelihoods be transformed to
```

Comment:
I'm now wondering if we should rename the function `log_likelihood` and remove this argument as in `offspring_ll`, especially if we eventually want to wrap this by an S3 function for `epichains` or related objects with the standard name `logLik`. The only difference is the `exp` call at the end but the user could also really do that themselves externally.

It would have the benefit that the mentions of "log-likelihood/likelihood" would be shortened to "log-likelihood".

Reply:
It's a good consideration. I will convert this into an issue and merge this feature first, then we can brainstorm how it further. I am wary of overshooting this PR again.

Reply:
I agree - good call.